src/uwtools/drivers/forecast.py

```diff
@@ -16,74 +16,55 @@
 
 class FV3Forecast(Driver):
     """
-    Concrete class to handle UFS Short Range Weather app forecasts.
-
-    FV3 (ATM-only) mode.
+    A driver for the FV3 forecast model.
     """
 
-    @property
-    def schema(self) -> str:
-        """
-        The schema to validate the config file against.
-        """
-        with resources.as_file(resources.files("uwtools.resources")) as path:
-            return (path / "FV3Forecast.jsonschema").as_posix()
+    # Public methods
 
-    def requirements(self):
+    def batch_script(self) -> None:
         """
-        Recursively parse config and platform files to determine and fill in any dependencies.
+        Write to disk, for submission to the batch system, a script to run FV3.
         """
 
-    def resources(self):
+    @staticmethod
```

Comment:
One of several methods that were reformulated as static methods, so that they can be called without first instantiating an `FV3Forecast` object.